src/InvariantTheory/affine_algebra.jl

```diff
@@ -0,0 +1,249 @@
+################################################################################
+#
+#  Presentation as affine algebra
+#
+################################################################################
+
+@doc Markdown.doc"""
+    affine_algebra(IR::InvRing;
+      algo_gens::Symbol = :default, algo_rels::Symbol = :groebner_basis)
+
+Given an invariant ring `IR` with underlying graded polynomial ring, say `R`,
+return a graded affine algebra, say `A`, together with a graded algebra
+homomomorphism `A` $\rightarrow$ `R` which maps `A` isomorphically onto `IR`.
+
+!!! note
+    If a system of fundamental invariants for `IR` is already cached, the function
+    makes use of that system. Otherwise, such a system is computed and cached first.
+    The algebra `A` is graded according to the degrees of the fundamental invariants,
+    the modulus of `A` is generated by the algebra relations on these invariants, and
+    the algebra homomomorphism `A` $\rightarrow$ `R` is defined by sending the `i`-th
+    generator of `A` to the `i`-th fundamental invariant.
+
+# Optional arguments
+Using the arguments `:king` or `:primary_and_secondary` for `algo_gens` selects
+the algorithm for the computation of the fundamental invariants (see
+[`fundamental_invariants`](@ref) for details).
+The argument `:groebner_basis` or `:linear_algebra` for `algo_rels` controls
+which algorithm for the computation of the relations between the fundamental
+invariants is used. With `:groebner_basis`, the relations are computed via the
+standard computation of a kernel of a morphism between multivariate polynomial
+rings. The option `:linear_algebra` uses an algorithm by Kemper and Steel
+[KS99, Section 17.5.5] to compute the relations without the use of Groebner
```

Comment:
Could you perhaps change this to use `@cite` (grep through the `docs` dir to see examples; the bib file already contains KS99)

Reply:
Yes, thanks, I totally forgot that.